package.json

```diff
@@ -20,7 +20,7 @@
     "js-yaml": "^3.14.0",
     "mocha-junit-reporter": "^2.0.0",
     "sauce-testrunner-utils": "^0.1.1",
-    "saucectl": "^0.23.2",
+    "saucectl": "^0.31.2",
```

Comment:
I don't think saucectl is required in runner anymore. It should be removed.

Reply:
@tianfeng92 I'll move saucectl to `devDependencies` so that it's not included in the runner itself, but is still available to run saucectl tests

Reply:
It could be moved to dev dependency, so it will be available for testing ?